packages/app/src/components/navigation/links/index.tsx

```diff
@@ -38,7 +38,9 @@ const links: {
 ];
 
 type Props = BaseProps;
-const Links: React.FC<Props> = ({ className = '', on }) => {
+const Links: React.FC<Props> & {
+  renewal: React.FC<Props>;
```

Comment:
What is this used for?

Reply:
This is for using the component like this:
```tsx
<Links />
<Links.renewal />
```

Reply:
Thanks!